lxc/completion.go

```diff
@@ -8,6 +8,7 @@ import (
 	"github.com/spf13/cobra"
 
 	"github.com/canonical/lxd/lxd/instance/instancetype"
+	"github.com/canonical/lxd/lxd/node"
```

Comment:
You won't be able to import this package into the client unfortunately. That's why the build is failing.